app/code/core/Mage/Core/Helper/UnserializeArray.php

```diff
@@ -26,11 +26,13 @@ class Mage_Core_Helper_UnserializeArray
      * @param string $str
      * @return array
      * @throws Exception
+     * @SuppressWarnings("PHPMD.ErrorControlOperator")
      */
     public function unserialize($str)
     {
         try {
-            $result = unserialize($str, ['allowed_classes' => false]);
+            $str = is_null($str) ? '' : $str;
```

Comment:
@sreichel In my [test](https://onlinephp.io/c/98cb5), line 34 is redundant. Why not remove it?

Reply:
Not tested again, but w/o it you are passing null to unserialize.

Please revome it from PR and lets see what happens.

Reply:
Unit tests on php8.3 

> PHP Deprecated:  unserialize(): Passing null to parameter #1 ($data) of type string is deprecated in /home/runner/work/magento-lts/magento-lts/app/code/core/Mage/Core/Helper/UnserializeArray.php on line 33